x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/search_source_expression_form.tsx

```diff
@@ -289,7 +301,7 @@ export const SearchSourceExpressionForm = (props: SearchSourceExpressionFormProp
         onChangeSelectedValue={onChangeSizeValue}
       />
       <EuiSpacer size="s" />
-      <TestQueryRow fetch={onTestFetch} hasValidationErrors={false} />
+      <TestQueryRow fetch={onTestFetch} copyQuery={onCopyQuery} hasValidationErrors={false} />
```

Comment:
Headsup: I am doing some refactoring in https://github.com/elastic/kibana/pull/134763/files#diff-ad63479b1020032a8cfa91ab84433536f84691788649cbe4c4516a224c6c8a09R137 to unify the view.

Reply:
Thanks for the heads up! Any ideas on how we might handle this once `rule_common_expressions.tsx` is introduced? Also this work is mostly done other than needing UX input and code reviews, but do you think we should avoid merging it until your refactoring is done?

Reply:
We can merge in any order.

Reply:
Sounds good. I just realized this will need some more work anyways since it doesn't work in Safari right now.